WPThemeReview/Tests/Privacy/NoUrlShortenersUnitTest.inc

```diff
@@ -0,0 +1,11 @@
+<?php
```

Comment:
As this sniff is supposed to also check JS and CSS, test case files for both ought to be added.